bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -491,6 +491,41 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+
+	<thing-type id="airqualitysensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz"/>
+		</supported-bridge-type-refs>
+		<label>Air quality Sensor</label>
+		<description>A air quality sensor</description>
```

Comment:
Texts must be updated in **src/main/resources/OH-INF/i18n/deconz.properties** also.

Reply:
Translations are done.